utils/test/Configuration.py

```diff
@@ -39,7 +39,7 @@
 militaryDataPath = os.path.normpath(os.path.join(currentPath, r"../../testdata/"))
 militaryInputDataGDB = os.path.normpath(os.path.join(militaryDataPath, "MilitaryToolsTestData.gdb"))
 militaryResultsGDB = os.path.normpath(os.path.join(militaryDataPath, "Results.gdb"))
-militaryScratchGDB = None
+militaryScratchGDB = os.path.normpath(os.path.join(currentPath, "scratch.gdb"))
```

Comment:
Do you intend this to go in the folder: `military-tools-geoprocessing-toolbox\utils\test\scratch.gdb` - if so you may want to add a gitignore to that folder to ignore this.